install/image-references

```diff
@@ -26,7 +26,7 @@ spec:
   - name: pod
     from:
       kind: DockerImage
-      name: quay.io/openshift/pod:v4.0
+      name: quay.io/tectonicshift/pod:42
```

Comment:
what is this image?

we need to use the pod image from the release payload.

Reply:
This image doesn't exist and we aren't actually using it - we're really using the `openshift/pod:v4.0` image still.  The reason to add this new dummy image is to help "ratchet" the change here so we can land [the installer PR](https://github.com/openshift/installer/pull/1292).

Yes, it is *very* confusing...

Reply:
See https://github.com/openshift/machine-config-operator/pull/471#issuecomment-467153179